src/index.ejs

```diff
@@ -17,12 +70,17 @@
     <% } %>
   </head>
   <body>
-    <!-- Initial drag area (for Linux and Windows). -->
-    <div id="init-drag-region" style="position: absolute;-webkit-app-region: drag;left: 0;right: 0;top: 0;height: 25px;"></div>
```

Comment:
Where did this go?

Reply:
Add a drag area so that we can drag this app to other place on the screen. but it's no need to use, we add an init loading page to instead it. we can also drag this app when the loading page is shown.